### PR TITLE
Daml Finance: Adding settlement-time section

### PR DESCRIPTION
**Pull request by johan-da**
FIXES https://github.com/digital-asset/daml-finance/issues/1201

### CODE REVIEW

File: docs/3.0.0/docs/daml-finance/concepts/settlement.rst [hunk 1]
```diff
@@ -218,6 +218,44 @@ of instructions atomically. Execution will fail if any of the
 :ref:`Instructions <module-daml-finance-settlement-instruction-87187>` is not fully allocated
 / approved, or if the transfer is unsuccessful.
 
+Settlement Time
+===============
+
+The settlement time for financial transactions can vary based on the market and the type of
+securities involved. Typically, settlement periods are denoted as T+1, T+2, or T+3, indicating the
+transaction date plus one, two, or three business days, respectively. Additionally, some markets may
+offer real-time settlement options. It's also common for certain trades between parties to have
```
Comment (carrielaben-da) [inline docs/3.0.0/docs/daml-finance/concepts/settlement.rst]:
```suggestion
transaction date plus one, two, or three business days, respectively. Some markets may also
offer real-time settlement options. It's also common for certain trades between parties to have
```

File: docs/3.0.0/docs/daml-finance/concepts/settlement.rst [hunk 2]
```diff
@@ -218,6 +218,44 @@ of instructions atomically. Execution will fail if any of the
...
+unique, mutually agreed-upon settlement periods.
+
+The :ref:`Batch <module-daml-finance-settlement-batch-95573>` and
+:ref:`Instruction <module-daml-finance-settlement-instruction-87187>`
+implementations are designed to allow for the optional setting of a preferred settlement time, yet
+they do not incorporate any mechanisms that enforce a specific settlement time. This design choice
+offers several advantages:
```
Comment (carrielaben-da) [inline docs/3.0.0/docs/daml-finance/concepts/settlement.rst]:
```suggestion
implementations are designed to allow the optional setting of a preferred settlement time, without any mechanisms that enforce a specific settlement time. This design choice
offers several advantages:
```

**Review by carrielaben-da — APPROVED** [review]
Just a couple of suggestions; looks good.

**Review by markus-da — APPROVED** [review]
LGTM

**Pull request merged**